### PR TITLE
swapped out media copy button for design-system button

### PR DESCRIPTION
**Pull request by d88naimi**
Swapped out media copy button for design-system button, cleaned up border radius and styles for Copy button.

Before
Screen Shot 2021-09-08 at 11 58 32 AM

After
Screen Shot 2021-09-08 at 11 58 44 AM

### CODE REVIEW

**Comment by shrunyan** [comment]
We should implement this like the other usage of `CopyButton`

**Comment by d88naimi** [comment]
V2 not using new `CopyInput` component 
PR: https://github.com/zesty-io/design-system/pull/152

**Comment by shrunyan** [comment]
> We should implement this like the other usage of `CopyButton`

What I meant by this was we should use the `CopyButton` as it is designed, without any modification. e.g. wrapping it around an input. The value does not need to be in an input as it can not be edited.

**Comment by d88naimi** [comment]
Removed input entirely and using  CopyButton component

default
Screen Shot 2021-09-13 at 11 49 52 AM


Hover
Screen Shot 2021-09-13 at 11 50 04 AM

File: src/apps/media/src/app/components/MediaDetailsModal.js [hunk 1]
```diff
@@ -150,7 +130,7 @@ export const MediaDetailsModal = memo(function MediaDetailsModal(props) {
         </div>
       </ModalContent>
       <ModalFooter className={shared.ModalFooter}>
-        <Button type="cancel" onClick={props.onClose}>
```
Comment (shrunyan) [inline src/apps/media/src/app/components/MediaDetailsModal.js:153]: Shouldn't the type be `cancel`? Is the design-system styling incorrect?

**Review by shrunyan — CHANGES_REQUESTED** [review]
- The media ZUID should also use a `CopyButton`.
- Test a long file path which would cause the button to overflow

File: src/apps/media/src/app/components/MediaDetailsModal.js (hunk 1 above)
Comment (d88naimi) [reply src/apps/media/src/app/components/MediaDetailsModal.js:153]: fixed and made a pr change

Screen Shot 2021-09-14 at 11 17 08 AM

**Review by shrunyan — APPROVED** [review]

**Pull request merged**